### ISSUE TITLE
Can you provide the ability to train "custom data"??

### ISSUE THREAD

**Issue by NaviDSX** [intel-isl/MiDaS#67]
It would be useful to add modules or notes for transfer learning so that we can use it forward.

**Comment by ranftlr** [comment]
There are currently no plans to release the training pipeline. 

You can find the loss here:

https://gist.github.com/ranftlr/1d6194db2e1dffa0a50c9b0a9549cbd2

The multi-objective optimizer can also be found here:

https://github.com/intel-isl/MultiObjectiveOptimization

**Comment by JohnG0024** [comment]
@ranftlr Hi, any update on the plans to release the training code yet?
